src/order/order_dual.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2020 Johan Commelin, Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Damiano Testa
+-/
+import data.subtype
+import data.prod
+import data.equiv.basic
+import data.finset.basic
+--import data.finsupp.basic
+--import order.lattice
```

Comment:
Can you get away with this?
```suggestion
import order.basic
import data.equiv.basic
```

Reply:
Yes!  Importing order.basic and data.equiv.basic is enough for order_dual!

Reply:
I pushed the change from my own computer, instead of committing your suggestion here.